frontend/src/metabase/css/components/header.css

```diff
@@ -67,7 +67,6 @@
   color: white;
   border: none;
   font-size: 1em;
-  text-transform: capitalize;
```

Comment:
@kdoh does removing this make sense?

Reply:
@salsakran Yup. It makes sense in the context of the other change. Basically going from `One Two` to `One two` and making sure CSS doesn't overwrite that.

Reply:
How many other strings does this affect?